src/libaktualizr/utilities/utils.cc

```diff
@@ -642,6 +643,7 @@ boost::filesystem::path Utils::absolutePath(const boost::filesystem::path &root,
   return (root / file);
 }
 
+#if (!defined(ANDROID) || __ANDROID_API__ >= 28)
 std::vector<boost::filesystem::path> Utils::glob(const std::string &pat) {
```

Comment:
`Utils::glob` can be deleted or replaced with cross-platform implementation.

Reply:
It is also could be allowed once we upgrade API from 27 to 28.